Sources/Player/UserInterface/ProgressTracker.swift

```diff
@@ -138,6 +152,18 @@ public final class ProgressTracker: ObservableObject {
         guard let timeRange, let progress else { return nil }
         return timeRange.start + CMTimeMultiplyByFloat64(timeRange.duration, multiplier: Float64(progress))
     }
+
+    private func pausePlaybackIfNeeded(with player: Player?) {
+        guard let player, player.playbackState == .playing else { return }
+        player.pause()
+        wasPaused = true
```

Comment:
```suggestion
        wasPaused.toggle()
```

Reply:
Probably a bit safer to force the value.